app/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,31 @@
+#{$all-button-inputs},
+button {
+  @include appearance(none);
+  -webkit-font-smoothing: antialiased;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, cursor, display, font-family, font-size, -webkit-font-smoothing, font-weight, line-height, padding, text-decoration, user-select, vertical-align, white-space